### PR TITLE
Adding logic to CI testing to check ES for workload results

### PR DESCRIPTION
**Pull request by dry923**
Added check_es python script that will check an ES instance for a given uuid and index. This is used to help validate that the tests ran and indexed properly. This was only added to workloads that utilize snafu.
Additionally, updated the backpack CI test to run as a privileged container with a service account. This way we can ensure we check as many of the functions of the pod as possible. However, I did have to add in a sleep after the workload was complete as ES seemed to take some time to process the data and make it available to check via the check_es script.

### CODE REVIEW

**Comment by dry923** [comment]
Note the backpack test will fail until https://github.com/cloud-bulldozer/scribe/pull/26 is merged

File: resources/backpack_role.yaml [hunk 1]
```diff
@@ -107,6 +107,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
```
Comment (aakarshg) [inline resources/backpack_role.yaml:111]: curious why is this needed ?

File: tests/test_backpack.sh [hunk 2]
```diff
@@ -21,23 +21,22 @@ function functional_test_backpack {
...
-      echo "Found stockpile.json on "$pod
-    else
-      echo "Could not find stockpile.json on "$pod
-      echo "Backpack test: Fail"
-      exit 1
-    fi
-  done
-
-  echo "Backpack test: Success"
+  indexes="cpu_vulnerabilities-metadata cpuinfo-metadata dmidecode-metadata k8s_configmaps-metadata k8s_namespaces-metadata k8s_nodes-metadata k8s_pods-metadata lspci-metadata meminfo-metadata sysctl-metadata"
```
Comment (aakarshg) [inline tests/test_backpack.sh:32]: this would mean everytime we add a new scribe module, we'll have to update this... im okay with it, but shouldn't backpack be tested outside of ripsaw? we are still testing its integration with ripsaw ( we've to wait for benchmark pods to come up, which in turn waits on backpack to finish )

File: resources/backpack_role.yaml (hunk 1 above)
Comment (dry923) [reply resources/backpack_role.yaml:111]: It was not successfully running the node collection due to an api restriction. Adding the coordination api group resolved it.

File: tests/test_backpack.sh (hunk 2 above)
Comment (dry923) [reply tests/test_backpack.sh:32]: We already have a test backpack script/cr in place. This was just adding to the robustness of the test. Additionally, this would ensure that errors related to the backpack role/permissions/etc get caught.

File: tests/check_es.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import argparse
+import elasticsearch
+import sys
+
+def _check_index(server,port,uuid,index):
```
Comment (jtaleric) [inline tests/check_es.py:18]: /me waits for someone to mention TLS ;)

**Comment by dry923** [comment]
Updated to include ssl checking for ES. @aakarshg can you confirm that looks correct to you?

**Comment by aakarshg** [comment]
> Updated to include ssl checking for ES. @aakarshg can you confirm that looks correct to you?

this LGTM
